build-logic-commons/commons/src/main/kotlin/common.kt

```diff
@@ -20,6 +20,5 @@ import org.gradle.jvm.toolchain.JvmVendorSpec
 fun JavaPluginExtension.configureJavaToolChain() {
     toolchain {
         languageVersion.set(JavaLanguageVersion.of(11))
-        vendor.set(JvmVendorSpec.ADOPTIUM)
```

Comment:
I don't think this should be removed.

Reply:
@hegyibalint just removed this on release7x: https://github.com/gradle/gradle/commit/1d3e8ed9d6d4bd855962bf60e2aaf82b4798a91b

Reply:
Well, that removal is a mistake. It looks like the intent was to remove the conditional, not the vendor. The vendor should stay.

Reply:
OK, I put that back in here https://github.com/gradle/gradle/pull/23243

Reply:
Yep, should be like now on `master`: https://github.com/gradle/gradle/blob/9842b0fa16ad451a449b88bde9e8647d3da84756/build-logic-commons/commons/src/main/kotlin/common.kt#L23

Do we need to change this on `release7x` as well then?